jerry-core/parser/js/common.c

```diff
@@ -74,6 +76,7 @@ util_print_number (ecma_number_t num_p) /**< number to print */
   lit_utf8_byte_t str_buf[ECMA_MAX_CHARS_IN_STRINGIFIED_NUMBER];
   lit_utf8_size_t str_size = ecma_number_to_utf8_string (num_p, str_buf, sizeof (str_buf));
   str_buf[str_size] = 0;
+  JERRY_UNUSED (str_buf); // potentially
```

Comment:
We will forget adding these macros. Just like asserts, logging should reference its arguments.